### PR TITLE
adds VisuallyHiddenText component, fixes #51

### LINKED ISSUE #51: Put VisuallyHiddenText component into Psammead
**Is your feature request related to a problem? Please describe.**
Note: blocked on #12 and https://github.com/BBC-News/simorgh/issues/975.

Corresponding 'pull from Psammead' issue in Simorgh: https://github.com/BBC-News/simorgh/issues/996.

**Describe the solution you'd like**
A PR in Psammead, which adds VisuallyHiddenText component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Unit tests (if any) will be transferred from Simorgh.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #51

adds VisuallyHiddenText component

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by pjlee11 — DISMISSED** [review]
Nice work

File: packages/components/psammead-visually-hidden-text/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Psammead Headings Changelog
```
Comment (dr3) [inline packages/components/psammead-visually-hidden-text/CHANGELOG.md]: The whitespace on this table is a bit crazy and could probably be improved

File: packages/components/psammead-visually-hidden-text/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Visually Hidden Text [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=VisuallyHiddenText)
```
Comment (dr3) [inline packages/components/psammead-visually-hidden-text/README.md]: This line should be updated to match the other packages

File: packages/components/psammead-visually-hidden-text/CHANGELOG.md (hunk 1 above)
Comment (dr3) [reply packages/components/psammead-visually-hidden-text/CHANGELOG.md]: Also it says Psammead Headings

**Review by dr3 — CHANGES_REQUESTED** [review]
Just a few things

File: packages/components/psammead-visually-hidden-text/README.md (hunk 2 above)
Comment (bcmn) [reply packages/components/psammead-visually-hidden-text/README.md]: Worth noting none of the other packages have Storybook in the Readme, which looks like an accident. 

```suggestion
# psammead-visually-hidden-text &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-visually-hidden-text.svg)](https://www.npmjs.com/package/@bbc/psammead-visually-hidden-text) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request) [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](https://simorghstorybook.now.sh/?selectedKind=VisuallyHiddenText)
```

Something like this? Noisy. 🔊 🙉

**Review by bcmn — APPROVED** [review]

**Review by pjlee11 — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**